### DISCUSSION TITLE
How to stop socketio client from printing in the terminal?

### DISCUSSION THREAD

**Discussion by gaurav2001trader** [miguelgrinberg/python-socketio#1063] [Q&A]
How to stop socketio client from printing in the terminal? For example when we receive on_message event the socketio client print the received data on terminal.

**Comment by miguelgrinberg** [comment]
@gaurav2001trader As long as you don't enable logging you shouldn't see anything. If you are enabling logging, then you have to configure the logging subsystem to suppress the logs that you do not want to see. If you don't show any code it is impossible for me to provide more detailed advice.

**Reply by gaurav2001trader** [reply]
Does printing in the vs code terminal also count as logging?
And also what is the difference between logger and engine io logger?

**Reply by miguelgrinberg** [reply]
This package knows nothing of the VS Code terminal that you are using. You are still not showing me anything, so I cannot really tell you anything new.

> And also what is the difference between logger and engine io logger?

The `logger` option controls the logging of Socket.IO protocol traffic and details. The `engineio_logger` option controls logging of lower level Engine.IO traffic.

**Reply by gaurav2001trader** [reply]
Understood , Thankyou.
